library/Icingadb/Widget/ItemList/BaseDowntimeListItem.php

```diff
@@ -66,7 +66,7 @@ protected function init()
         $this->isActive = $this->item->is_in_effect
             || $this->item->is_flexible && $this->item->scheduled_start_time <= $this->currentTime;
 
-        $until = ($this->isActive ? $this->endTime : $this->startTime) - $this->currentTime;
+        $until = (int) ($this->isActive ? $this->endTime : $this->startTime) - $this->currentTime;
```

Comment:
The error occurred on line no. 71. I suspect it's been the modulo operation that caused it. Can't you just cast `$until` to int there? Then the date formatting would still get the float, might be useful :thinking: